### PR TITLE
Release tools

### PR DESCRIPTION
**Pull request by Informatic**
`npm version minor` should:
* bump minor version in `package.json`
* copy the version over to `appinfo.json`
* create new version commit
* tag said commit

Also `deploy` target is now installing properly named .ipk from a local directory.

### CODE REVIEW

**Comment by mariotaku** [comment]
This is cool!

File: package.json [hunk 1]
```diff
@@ -17,7 +17,9 @@
         "build-local:frontend": "enyo pack",
         "build-local:service": "webpack",
         "build-local:package": "ares-package dist/ -e enyo-ilib",
-        "build-and-launch": "npm-run-all --serial build-local deploy launch"
+        "build-and-launch": "npm-run-all --serial build-local deploy launch",
+        "version": "node tools/sync-version.js && git add appinfo.json",
+        "postversion": "git push --tags"
```
Comment (mariotaku) [inline package.json:22]: Do we need automatic push? I prefer to leave this step to be done manually.

Comment (Informatic) [reply package.json:22]: Not really, I can remove that.

**Review by mariotaku — APPROVED** [review]
LGTM

**Comment by Informatic** [comment]
I'll need to rework webpack service build to copy over "main" package.json (or create a dummy file) during build, since nested `package.json` files make `eslint` angry...

**Pull request merged**